pkg/pillar/types/volumetypes.go

```diff
@@ -0,0 +1,173 @@
+		Infof("Volume config create")
+}
+
+// LogModify :
+func (config VolumeConfig) LogModify(old interface{}) {
+	logObject := base.EnsureLogObject(base.VolumeConfigLogType, config.DisplayName,
+		config.VolumeID, config.LogKey())
+
+	oldConfig, ok := old.(VolumeConfig)
+	if !ok {
+		log.Errorf("LogModify: Old object interface passed is not of VolumeConfig type")
+	}
+	if oldConfig.ContentID != config.ContentID ||
+		oldConfig.MaxVolSize != config.MaxVolSize ||
+		oldConfig.GenerationCounter != config.GenerationCounter {
+
+		logObject.CloneAndAddField("content-id", config.ContentID).
+			AddField("maxVolSize", config.MaxVolSize).
+			AddField("generationCounter", config.GenerationCounter).
+			AddField("old-content-id", oldConfig.ContentID).
+			AddField("old-maxVolSize", oldConfig.MaxVolSize).
+			AddField("old-generationCounter", oldConfig.GenerationCounter).
+			Infof("Volume config modify")
+	}
+}
+
+// LogDelete :
+func (config VolumeConfig) LogDelete() {
+	logObject := base.EnsureLogObject(base.VolumeConfigLogType, config.DisplayName,
+		config.VolumeID, config.LogKey())
+	logObject.CloneAndAddField("content-id", config.ContentID).
+		Infof("Volume config delete")
+
+	base.DeleteLogObject(config.LogKey())
+}
+
+// LogKey :
+func (config VolumeConfig) LogKey() string {
+	return string(base.VolumeConfigLogType) + "-" + config.Key()
+}
+
+// VolumeStatus is response from volumemgr about status of volumes
+type VolumeStatus struct {
+	VolumeID                uuid.UUID
+	ContentID               uuid.UUID
+	VolumeContentOriginType zconfig.VolumeContentOriginType
+	MaxVolSize              uint64
+	ReadOnly                bool
+	GenerationCounter       int64
+	DisplayName             string
+	State                   SwState
+	Progress                uint   // In percent i.e., 0-100
+	FileLocation            string // Location of filestystem
+	VolumeCreated           bool   // Done aka Activated
+	ContentFormat           zconfig.Format
+	LastUse                 time.Time
+	PreReboot               bool // Was volume last use prior to device reboot?
+
+	ErrorAndTimeWithSource
+}
+
+// Key is volume UUID which will be unique
+func (status VolumeStatus) Key() string {
+	return fmt.Sprintf("%s.%d", status.VolumeID.String(), status.GenerationCounter)
+}
+
+// IsContainer will return true if content tree attached
+// to the volume is of container type
+func (status VolumeStatus) IsContainer() bool {
+	if status.ContentFormat == zconfig.Format_CONTAINER {
+		return true
+	}
+	return false
+}
+
+// LogCreate :
+func (status VolumeStatus) LogCreate() {
+	logObject := base.NewLogObject(base.VolumeStatusLogType, status.DisplayName,
+		status.VolumeID, status.LogKey())
+	if logObject == nil {
+		return
+	}
+	logObject.CloneAndAddField("contentID", status.ContentID).
+		AddField("maxVolSize", status.MaxVolSize).
+		AddField("state", status.State).
+		AddField("progress", status.Progress).
+		AddField("fileLocation", status.FileLocation).
```

Comment:
Not camelcase.

Reply:
Resolved.

Reply:
The diffs in front of me show fileLocation in camelCase. Does github show me something old?